### ISSUE TITLE
Most overconnected issue list in issue tree

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#1101]
It'd really help to have a hard list, up front, of which issues were most over-connected

Blocked by #318; blocked by #826

**Comment by StoneCypher** [comment]
Superseded by the v6→v16 era program — this tracking/meta issue is absorbed into the structured plan (the version umbrellas #1401–#1411 and the FSL Roadmap project board now carry the work). Closing per the approved triage ledger (`notes/superpowers/eras/dispositions/fsl-triage.md`).
